Tests/BugsnagSessionTrackerTest.m

```diff
@@ -63,9 +62,6 @@ - (void)testStartNewAutoCapturedSession {
     XCTAssertNotNil(session);
     XCTAssertNotNil(session.sessionId);
     XCTAssertTrue([[NSDate date] timeIntervalSinceDate:session.startedAt] < 1);
-    XCTAssertNil(session.user.name);
-    XCTAssertNil(session.user.userId);
-    XCTAssertNil(session.user.emailAddress);
```

Comment:
Do these have sensible values now or something? I'd expect that they would still be nil.

Reply:
Persisted user details caused these to fail.  These are now cleared out in setup(). The AssertNils have been restored.